src/viking.js

```diff
@@ -1,11 +1,91 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor (health ,strength){
+        this.health= health,
+        this.strength= strength
+    }
+
+    attack(){
+        return this.strength;
+    }
+    receiveDamage(damage){
+        this.health = this.health - damage;
```

Comment:
**A tip:** You could use an assignment operator here to do the same thing. This way you won't need to write the ```this.health``` twice.  Check suggestion below
```suggestion
        this.health -= damage;
```